frontend/src/bundles/landing/components/feedbacks-part/feedbacks-part.tsx

```diff
@@ -10,20 +17,53 @@ type Properties = {
     }[];
 };
 
+const bulletActive = styles.bulletActive;
+const bulletClass = styles.bulletClass;
+const horizontalClass = styles.horizontalClass;
+
+const paginationOptions = {
+    bulletClass: bulletClass,
+    bulletActiveClass: bulletActive,
+    horizontalClass: horizontalClass,
+    clickable: true,
+};
+
+const breakpoints = {
+    992: {
+        slidesPerView: 3,
+        spaceBetween: 35,
+    },
+    768: {
```

Comment:
Add an enum with these values

Reply:
Resolved in [0b210a5](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/189/commits/0b210a5e85aba7993ed3dbd680ec2d4640f94eab)